linux/SystemdMeter.c

```diff
@@ -316,12 +318,12 @@ static int zeroDigitColor(unsigned int value) {
 
 static int valueDigitColor(unsigned int value) {
    switch (value) {
-      case 0:
-         return CRT_colors[METER_VALUE_NOTICE];
-      case INVALID_VALUE:
-         return CRT_colors[METER_VALUE_ERROR];
-      default:
-         return CRT_colors[METER_VALUE];
+   case 0:
+      return CRT_colors[METER_VALUE_NOTICE];
+   case INVALID_VALUE:
+      return CRT_colors[METER_VALUE_ERROR];
+   default:
+      return CRT_colors[METER_VALUE];
```

Comment:
The indent style of `case` statements inside `switch` are not consistent even after this PR.
Do `case` statements indent one level after `switch`...
```c
switch (value) {
   case 0:
      // Do something
      break;
   case 1:
      // Do something else
      break;
   default:
      // Yet something else
}
```
Or do they stay the same level as the `switch`...
```c
switch (value) {
case 0:
   // Do something
   break;
case 1:
   // Do something else
   break;
default:
   // Yet something else
}
```